frontend/packages/console-app/src/components/nodes/NodeTerminal.tsx

```diff
@@ -0,0 +1,109 @@
+import * as React from 'react';
+import { Firehose, FirehoseResult, LoadingBox } from '@console/internal/components/utils';
+import { NodeKind, PodKind } from '@console/internal/module/k8s';
+import { PodExecLoader } from '../../../../../public/components/pod';
+import { PodModel } from '../../../../../public/models';
+import { k8sCreate, k8sKill } from '../../../../../public/module/k8s';
+
+type NodeTerminalInnerProps = {
+  obj?: FirehoseResult<PodKind>;
+};
+
+type NodeTerminalProps = {
+  obj: NodeKind;
+};
+
+const getDebugPod = (node) => {
+  return {
+    kind: 'Pod',
+    apiVersion: 'v1',
+    metadata: {
+      generateName: `${node.metadata.name}-debug-`,
+      namespace: 'default',
+      creationTimestamp: null,
+      annotations: {
+        'debug.openshift.io/source-container': 'container-00',
+        'debug.openshift.io/source-resource': `/v1, Resource=nodes/${node.metadata.name}`,
+      },
+    },
+    spec: {
```

Comment:
We might set `activeDeadlineSeconds` so that the pod doesn't stay running forever if we can't delete it (for instance, if the user kills the browser without leaving the Terminal tab).

Reply:
Agreed as I notice refreshing the browser doesn't delete the pod even with the addition of a window event listener.  @spadgett, what's a reasonable value?  It looks like there are several instances of `21600` in the code base.

Reply:
I've updated to [add `activeDeadlineSeconds`](https://github.com/openshift/console/pull/4780/files#diff-3b4082d1b9305f656b4b6f40d11d7240R51), but it doesn't make it in to the pod.  Not sure why.

Reply:
> I've updated to [add `activeDeadlineSeconds`](https://github.com/openshift/console/pull/4780/files#diff-3b4082d1b9305f656b4b6f40d11d7240R51), but it doesn't make it in to the pod. Not sure why.

This is fixed.  I believe removing `status` from the pod template made it work.  It does appear we have a bug on pod details where the value of `Active Deadline Seconds` does not display the correct value though.

<img width="1453" alt="Screen Shot 2020-03-23 at 4 33 27 PM" src="https://user-images.githubusercontent.com/895728/77360653-211cb080-6d24-11ea-9c84-e0f040f7db3a.png">

Reply:
> > I've updated to [add `activeDeadlineSeconds`](https://github.com/openshift/console/pull/4780/files#diff-3b4082d1b9305f656b4b6f40d11d7240R51), but it doesn't make it in to the pod. Not sure why.
> 
> This is fixed. I believe removing `status` from the pod template made it work. It does appear we have a bug on pod details where the value of `Active Deadline Seconds` does not display the correct value though.
> 
> <img alt="Screen Shot 2020-03-23 at 4 33 27 PM" width="1453" src="https://user-images.githubusercontent.com/895728/77360653-211cb080-6d24-11ea-9c84-e0f040f7db3a.png">

PR to resolve: https://github.com/openshift/console/pull/4801